src/css/styles.css

```diff
@@ -220,3 +220,12 @@ ins {
     font-weight: bold;
     margin-right: 0.5em;
 }
+
+.change-view-actions .btn-link {
+  border: none;
+  padding: 0;
+  font-size: inherit;
```

Comment:
Without removing the space and border, they do not line up with the other links in `.change-view-actions` (same with `vertical-align`).

Changing `display`, `line-height`, and `font-size`, don’t have any practical effect with the styles as they are, but they make any styling on `.change-view-actions` apply to these buttons in the same way that they apply to the other links. e.g. if you set `.change-view-actions { font-size: 1.2em; }` or similar, it would affect the links, but would not affect the buttons unless these `inherit` styles were here.